spec/services/search/search_service_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Search::SearchService, type: :service do
+  describe '#initialize' do
```

Comment:
For a class methods, it should be:

```suggestion
  describe '.new' do
```

See https://www.betterspecs.org/#describe